Makefile

```diff
@@ -19,7 +19,7 @@ BUILD_OUTPUT_DIRECTORY = $(BUILD_DIRECTORY)/out
 # ---------------------------------------------------------------------
 
 ELECTRON_VERSION = $(shell jq -r '.devDependencies["electron"]' package.json)
-NODE_VERSION = 6.10.3
+NODE_VERSION = 6.1.0
```

Comment:
It shouldn't matter, since the ABI is the same. The only advantage of running 6.10.3 is that it comes with the exact npm version that we're interested in, so we don't have to manually install it afterwards.